GoogleCloudPlatform/kubernetes-engine-samples
# Redis bump

## Context
Renovate bot had a PR open for bumping the redis version under hello-app-redis #254 

## What this PR does 
Bumps the redis version, and adds an argument to a function (client context). 

## To test
Ensure you can build the image, and replace with your image in the app-deployment.yaml files under manifests!
